lib/internal/streams/buffer_list.js

```diff
@@ -1,8 +1,10 @@
 'use strict';
 
 const {
+  StringPrototypeSlice,
   SymbolIterator,
   Uint8Array,
+  TypedArrayPrototypeSet,
```

Comment:
```suggestion
  TypedArrayPrototypeSet,
  Uint8Array,
```
nit: alphabetical order